src/sa_web/static/css/default.css

```diff
@@ -1766,6 +1766,9 @@ a.auth-inline {
   font-family: FontAwesome;
   overflow: hidden;
 }
+.place-story-bar h1 {
+  line-height: 1em;
+}
```

Comment:
Don't forget to check in the changes from the newly pre-compiled `default.css`! 

I think it needs to be updated with the newly added `vertical-align` attributes. 

Eventually this process will be automated, but for now, let's just remember to manually check in the changes on our default.css file.